sdk/core/src/CommonNetworkMember/NetworkMemberWithoutCognito.ts

```diff
@@ -0,0 +1,98 @@
+import { profile, KeysService } from '@affinidi/common'
+import { EventComponent } from '@affinidi/affinity-metrics-lib'
+import { SdkOptions } from '../dto/shared.dto'
+import { AffinidiCommonConstructor, IPlatformEncryptionTools } from '../shared/interfaces'
+import { ParametersValidator } from '../shared/ParametersValidator'
+import { randomBytes } from '../shared/randomBytes'
+import { getOptionsFromEnvironment, ParsedOptions } from '../shared/getOptionsFromEnvironment'
+import { BaseNetworkMember } from './BaseNetworkMember'
+
+type GenericConstructor<T> = new (password: string, encryptedSeed: string, options: ParsedOptions) => T
+type Constructor<T> = GenericConstructor<T> & GenericConstructor<NetworkMemberWithoutCognito>
+type AbstractStaticMethods = Record<never, never>
+type ConstructorKeys<T> = {
+  [P in keyof T]: T[P] extends new (...args: unknown[]) => unknown ? P : never
+}[keyof T]
+type OmitConstructor<T> = Omit<T, ConstructorKeys<T>>
+type DerivedTypeForOptions<TInstance> = Constructor<TInstance> &
+  AbstractStaticMethods &
+  OmitConstructor<typeof NetworkMemberWithoutCognito>
+type DerivedType<T extends DerivedType<T>> = DerivedTypeForOptions<InstanceType<T>>
+export type UniversalDerivedType = DerivedType<DerivedTypeForOptions<NetworkMemberWithoutCognito>>
+
+@profile()
+export abstract class NetworkMemberWithoutCognito extends BaseNetworkMember {
```

Comment:
Why do we need to add a suffix if class doesn't contain something? Why not "NetworkMember" and "NetworkMemberWithCognito"? It'll be too hard to use?

Reply:
If we had `NetworkMemberWithCognito` and `NetworkMember` it would look like `NetworkMemberWithCognito` extends `NetworkMember`?

These are two different classes extended from the same base; one handles scenarios when user management is required, and another handles scenarios where there should be no user management at all.

Maybe the current naming is questionable; do you have any ideas besides just plain `NetworkMember`?